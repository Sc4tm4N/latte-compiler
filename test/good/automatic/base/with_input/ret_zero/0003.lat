string happyface() {
  return ":D";
}

int main() {
  string s1 = readString(),
    s2 = readString(),
    s3 = readString(),
    s4 = readString(),
    s5 = readString(),
    s6 = readString(),
    s7 = readString(),
    s8 = s1 + s2 + s3 + s4 + s5 + s6 + s7 + happyface();

  printString(s8 + " " + happyface());
  return 0 ;
}