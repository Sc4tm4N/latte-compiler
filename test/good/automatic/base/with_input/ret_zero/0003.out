ala ma kota a kot ma ale:D :D
